### DISCUSSION TITLE
Add "Don't show again" option to user content disclaimer

### DISCUSSION THREAD

**Discussion by DanielPower** [ppy/osu#29400] [Ideas]
Currently, every time osu! is launched, the "Featured Artists" filter will be re-enabled. Attempting to disable it will result in a "User content disclaimer" window which the user much accept.

While this is a relatively minor complaint, as someone who plays osu! almost daily, I'd prefer to be able to suppress this warning, perhaps by checking a "Don't show again" box. I don't believe the repeated appearance of the disclaimers provides any added value to recurring players.

**Comment by Arhyved** [comment]
Instead of suppressing the message its better to save settings of the filter.
Every time i open the game i have to click 3 buttons: (unselect Featured Artist) (Mode: osu!) (Categories: Any). This is annoying.
